library/modules/Textures.cpp

```diff
@@ -109,7 +141,7 @@ std::vector<TexposHandle> slice_tileset(SDL_Surface* surface, int tile_px_w, int
                 surface->format->Bmask, surface->format->Amask);
             SDL_Rect vp{tile_px_w * x, tile_px_h * y, tile_px_w, tile_px_h};
             DFSDL_UpperBlit(surface, &vp, tile, NULL);
-            auto handle = Textures::loadTexture(tile);
+            auto handle = Textures::loadTexture(tile, reserved);
```

Comment:
if `reserved == true` we should do a pre-check before starting the loop to ensure that all tiles will fit in the remaining reserved buffer, and fall back to making them all dynamic if they won't all fit.

Reply:
and again it is hiden behaviour... does this fallback should be automatic?